### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by SeanChengN**

### CODE REVIEW

File: src/main/resources/assets/solcarrot/lang/zh_cn.lang [hunk 1]
```diff
@@ -17,15 +17,36 @@ command.solcarrot.foodlist.sync.success=食物清单已同步!
...
-message.solcarrot.desire.continues=品尝另外%s种食物来获得另外%s!
-message.solcarrot.desire.finished=你寻找独特食物的愿望终于得到满足.
+message.solcarrot.finished.chat=你寻找独特食物的愿望终于得到满足.
+message.solcarrot.finished.hotbar=你已经获得了%s. 你寻找独特食物的愿望终于得到满足.
 
 # Items
 item.solcarrot.food_book.name=食物清单
 
 # GUIs
 gui.solcarrot.food_book.title=我的食物日志
```
Comment (juliand665) [inline src/main/resources/assets/solcarrot/lang/zh_cn.lang:30]: This is no longer used; you can probably just remove it

**Comment by juliand665** [comment]
[Update has been uploaded; currently being processed by CurseForge.](https://minecraft.curseforge.com/projects/spice-of-life-carrot-edition/files/2686930)

Thanks for the help :)

File: src/main/resources/assets/solcarrot/lang/zh_cn.lang (hunk 1 above)
Comment (SeanChengN) [reply src/main/resources/assets/solcarrot/lang/zh_cn.lang:30]: Oh! I missed it. 
gui.solcarrot.food_book.stats=概要

Comment (juliand665) [reply src/main/resources/assets/solcarrot/lang/zh_cn.lang:30]: Ah, that's all right. I replaced it in the newest version; uploading soon.

**Pull request merged**